src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -530,20 +527,40 @@ public void deleteComponent(String project, String application, String context,
 
     @Override
     public void follow(String project, String application, String context, String component) throws IOException {
-        if (context != null) {
-            ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), true, envVars, command, "log", "-f");
-        } else {
-            ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, true, envVars, command, "log", "-f");
-        }
+        ExecHelper.executeWithTerminal(
+                this.project, WINDOW_TITLE,
+                createWorkingDirectory(context),
+                true,
+                envVars,
+                null,
+                null,
+                null,
+                command,
+                "log", "-f");
```

Comment:
@sbouchet, @jeffmaury, @lstocchi: please verify this fix for breaking changes in intellij-common